tests/integration/component_provider/nodejs/component-provider-host/provider/index.ts

```diff
@@ -1,5 +1,47 @@
 // Copyright 2025-2025, Pulumi Corporation.  All rights reserved.
 
-import { componentProviderHost } from "@pulumi/pulumi/provider/experimental";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
```

Comment:
Would this test still pass if you'd deleted `index.ts` and not `component.ts`? Guessing we always require an `index` for the `require` to work?

Reply:
No, it fails with `We failed to locate the entry point for your program`